deploy/crds/kieapp.crd.yaml

```
@@ -670,120 +671,516 @@ spec:
-                                description: Defines the interval for the background-validation
-                                  check for the jdbc connections.
-                                type: string
-                              connectionChecker:
-                                description: An org.jboss.jca.adapters.jdbc.ValidConnectionChecker
-                                  that provides a SQLException isValidConnection(Connection
-                                  e) method to validate if a connection is valid.
-                                type: string
-                              driver:
-                                description: Driver name to use. For example, mysql
-                                type: string
-                              exceptionSorter:
-                                description: An org.jboss.jca.adapters.jdbc.ExceptionSorter
-                                  that provides a boolean isExceptionFatal(SQLException
-                                  e) method to validate if an exception should be
-                                  broadcast to all javax.resource.spi.ConnectionEventListener
-                                  as a connectionErrorOccurred.
-                                type: string
-                              jdbcURL:
-                                description: Database JDBC URL. For example, jdbc:mysql:mydb.example.com:3306/rhpam
-                                type: string
-                              maxPoolSize:
-                                description: Sets xa-pool/max-pool-size for the configured
-                                  datasource.
-                                type: string
-                              minPoolSize:
-                                description: Sets xa-pool/min-pool-size for the configured
-                                  datasource.
-                                type: string
-                              password:
-                                description: External database password
-                                format: password
-                                type: string
-                              username:
-                                description: External database username
-                                type: string
-                            required:
-                            - driver
-                            - jdbcURL
-                            - password
-                            - username
-                            type: object
-                          size:
-                            description: Size of the PersistentVolumeClaim to create.
-                              For example, 100Gi
+                          allowExternalFileRegister:
+                            description: Allow download of external (remote) files
+                              into runtime. Default value is false
+                            type: boolean
+                          compEnable:
```

I think it would be better to not use short cut for component. And we should follow same pattern, e.g. next property is called `componentPartition`.
Do not merger suggestion and property name change needs to be reflected in all places.
```suggestion
                          componentEnable:
```

---

make sense, updated, thanks.